train.py

```diff
@@ -359,8 +345,6 @@
                    help='lower precision AMP dtype (default: float16)')
 group.add_argument('--amp-impl', default='native', type=str,
                    help='AMP impl to use, "native" or "apex" (default: native)')
-group.add_argument('--amp-opt-level', default='O1', type=str,
```

Comment:
https://github.com/moreh-dev/moreh-testbench/blob/main/yamls/nexus/tiny_set_acc_without_issues.yaml#L810
We need to remove O2 tests in our yamls?

Reply:
I just saw [this PR](https://github.com/moreh-dev/moreh-testbench/pull/197).

But why there's still fp16 tests?
In MAF, is it available  pytorch native amp? 
```
    amp_impl: native
    amp_dtype: float16
```

Reply:
> In MAF, is it available pytorch native amp?

Yes it is